src/App.css

```diff
@@ -0,0 +1,577 @@
+@import url("https://fonts.googleapis.com/css2?family=Merriweather+Sans:ital@1&display=swap");
+@import url(d:\Fiveerr ka kaam\awesome\src\fonts\style.css);
+body {
+  margin: 0%;
+  padding: 0%;
+  -webkit-box-sizing: border-box;
+          box-sizing: border-box;
+  background-color: #faf9fe;
```

Comment:
App.jsx must not serve any major styles (in App.js we setup store.. routes and kinf off)
it;s not a good idea to have a *css files to it

Reply:
Also, absolute imports shouldn't be allowed.
`
@import url(d:\Fiveerr ka kaam\awesome\src\fonts\style.css);
`
is going to make it crash for everyone having that line.